scene/gui/text_edit.cpp

```diff
@@ -867,11 +873,14 @@ void TextEdit::_notification(int p_what) {
 
 				// Ensure we at least use the font color.
 				Color current_color = !editable ? font_readonly_color : font_color;
+				if (draw_placeholder) {
+					current_color.a *= placeholder_alpha;
```

Comment:
I think it would be better to provide it as a dedicated theme color property (that can default to have 0.6 alpha).

Reply:
The same is used in the `LineEdit`, so if it's moved to the theme, it should be done for both.

Reply:
Let's do that as a follow-up PR for `LineEdit` and now `TextEdit` too.